__tests__/app.test.js

```diff
@@ -40,3 +40,37 @@ describe("GET - getCategories", () => {
     });
   });
 });
+
+describe("/api/reviews/:review_id", () => {
+  test("Get - status: 200 - responds with the first review ", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then((response) => {
+        // Arrange
+        const {review} = response.body
+        // Assert
+        expect(review.title).toBe("Agricola");
+        expect(review.designer).toBe("Uwe Rosenberg");
+        expect(review.owner).toBe("mallionaire");
+        expect(review.review_img_url).toBe(
+          "https://images.pexels.com/photos/974314/pexels-photo-974314.jpeg?w=700&h=700"
+        );
+        expect(review.category).toBe("euro game");
+        expect(review.created_at).toBe(
+          JSON.parse(JSON.stringify(new Date(1610964020514)))
+        );
+        expect(review.votes).toBe(1);
+        expect(typeof review.review_id).toEqual("number");
+      });
+  });
+  test("Get - status: 404 - responds with review not found", () => {
+    return request(app)
+      .get("/api/reviews/300")
+      .expect(404)
+      .then((response) => {
+        const { msg } = response.body;
+        expect(msg).toBe("review not found");
+      });
+  });
+});
```

Comment:
We should also have a status 400 for if the review doesn't only just not exist, but it will NEVER exist -> e.g. if the review ID is a string instead of a number